lib/broccoli/angular2-app.js

```diff
@@ -87,10 +87,16 @@ Angular2App.prototype.toTree = function () {
     destDir: 'vendor'
   });
 
+  var publicDirTree = new Funnel(path.join(sourceDir, 'public'), {
```

Comment:
Ah, it was my understanding it would be pulled from `src` and moved to `dist`. The issue wasn't very detailed though, so that was my fault for not asking.